projects/newthrone/index.js

```diff
@@ -1,11 +1,10 @@
 const { sumTokensExport } = require("../helper/unwrapLPs");
-const ADDRESSES= require("../helper/coreAssets.json");
 
 const GAME_CONTRACT = '0x61C3A357bc3ca51b80eCD36CB1Ae37e5465C6701'
 
 module.exports = {
-  methodology: 'WETH tokens in the Game contract, which represents the current balance of the game (total spent - total claimed).',
+  methodology: 'THRO tokens in the Game contract, which represents the current balance of the game (total spent - total claimed).',
   base: {
-    tvl: sumTokensExport({ owner: GAME_CONTRACT, tokens: [ADDRESSES.base.WETH]}),
+    tvl: sumTokensExport({ owner: GAME_CONTRACT, tokens: ['0x0f929C29dcE303F96b1d4104505F2e60eE795caC']})
```

Comment:
hi @ItsMeTazz we cant count THRO as tvl, can instead set tvl to 0 and count this under staking?

Reply:
Hey @g1nt0ki What would be the result of doing this? 

The TLDR is that players invest THRO, which is redistributed over time from the game contract. Our TVL represents the total of THRO in that game contract.

Reply:
yes, I understand, but we can count this investment as staking, not as tvl: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

Reply:
Okay, I don't know what the visual/numeral result of doing this is, but as long as we can track the total of THRO in the contract, I think we're good.

Reply:
@g1nt0ki How long does it usually take to be updated on the live website? Still seeing 0, wondering if it updates every X hours, or if it should be instantaneous. 

Thanks again for your time!

Reply:
did you toggle staking? 
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/99907941/533d3265-6edf-4065-b2c5-b12b7c1b6c90)